packages/cloud_firestore/android/src/main/java/io/flutter/plugins/firebase/cloudfirestore/CloudFirestorePlugin.java

```diff
@@ -60,6 +73,18 @@ private DocumentReference getDocumentReference(Map<String, Object> arguments) {
     return FirebaseFirestore.getInstance().document(path);
   }
 
+  private Transaction getTransaction(Map<String, Object> arguments) {
+    return transactions.get(getTransactionId(arguments));
+  }
+
+  private int getTransactionId(Map<String, Object> arguments) {
+    return (Integer) arguments.get("transactionId");
+  }
+
+  private long getTransactionTimeout(Map<String, Object> arguments) {
```

Comment:
I feel like a one-line private method that is only used in one place isn't really adding much in the way of code re-use and it would be better to just inline the lookup. The `getTransaction` method is ok, `getTransactionId` is iffy since it's only used in two places.

Reply:
Done, removed `getTransactionId` an d `getTransactionTimeout`